superset-frontend/src/pages/SavedQueryList/index.tsx

```diff
@@ -449,6 +453,12 @@ function SavedQueryList({
         input: 'search',
         operator: FilterOperator.AllText,
       },
+      {
+        Header: t('Description'),
+        id: 'description',
+        key: 'description',
+        operator: FilterOperator.AllText,
```

Comment:
The search filter should already be checking the description column, so is this really necessary?

Reply:
Instead we can rename the "Name" filter to be "Name and Description"

Reply:
it's actually searching all text columns (name, description, sql, label), so it can probably just be called "search"

https://github.com/apache/superset/blob/f5843fe588112c5c303215945a36fda0845c9e99/superset/queries/saved_queries/filters.py#L30

Reply:
@nytai I believe the ask is to show query description in a separate second column.

https://github.com/apache/superset/issues/12431#issuecomment-1310895547